src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -2941,10 +2997,18 @@ void Instruction::execute() {
       results[2] = operands[2].get<uint64_t>() + offset;
       break;
     }
+    case Opcode::AArch64_LDARB: {  // ldarb wt, [xn]
+      results[0] = memoryData[0].zeroExtend(1, 8);
```

Comment:
If this is being written to a `wt` register, shouldn't it be zero extended to `4` bytes not `8`?

Reply:
both w and x registers point to a 64-bit reg so needs to be extended to 8 in both cases